vulnerabilities/migrations/0034_package_package.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.7 on 2022-11-25 12:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vulnerabilities', '0033_alter_vulnerabilityseverity_scoring_system'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='package',
+            name='package',
+            field=models.CharField(blank=True, help_text='The Package URL for this package.', max_length=255),
```

Comment:
This should not be blank ever once migrated, so you would need a third migration once the data is updated.